source/help/settings/account-settings.rst

```diff
@@ -180,6 +180,10 @@ Teammate Name Display
 
 Configure how names are displayed in Mattermost: nickname, username or full name.
 
+Website Link Previews
+~~~~~~~~~~~~~~~~~~~~~~~~
+When available, the first web link in a message will show a preview of the website content below the message. This setting must be enabled by your System Admin.
```

Comment:
Propose linking to https://docs.mattermost.com/administration/config-settings.html#link-previews